### PR TITLE
Added displaying error messages in UI

### PR DESCRIPTION
**Pull request by w1gs**
This PR adds error messages in the extension UI. This should make it easier for new users to understand why a request has failed. This should also handle aborting correctly so if there are any errors, the UI features that depend on aborted connections (like resetting the state of the send button from the 'stop' state) also get updated. 

Examples:
CleanShot 2024-10-03 at 21 36 40@2x

CleanShot 2024-10-03 at 21 37 19@2x

CleanShot 2024-10-03 at 21 38 31@2x

CleanShot 2024-10-03 at 21 39 56@2x

CleanShot 2024-10-03 at 21 42 35@2x

### CODE REVIEW

File: extensions/void/src/common/sendLLMMessage.ts [hunk 1]
```diff
@@ -1,286 +1,408 @@
 import Anthropic from '@anthropic-ai/sdk';
```
Comment (Hieuuuu96) [inline extensions/void/src/common/sendLLMMessage.ts:1]: ehieu5102@gmail.com

File: extensions/void/src/common/sendLLMMessage.ts [hunk 2]
```diff
@@ -1,286 +1,408 @@
...
-
-
+		endpoint: string;
+		model: string;
+	};
+	whichApi: string;
+};
 
-type OnText = (newText: string, fullText: string) => void
+type OnText = (newText: string, fullText: string) => void;
```
Comment (Hieuuuu96) [inline extensions/void/src/common/sendLLMMessage.ts:32]: Screenshot_2024-10-06-01-50-16-93_40deb401b9ffe8e1df2f1cc5ba480b12

**Comment by andrewpareles** [comment]
What autoformatter are you using?

**Comment by w1gs** [comment]
> What autoformatter are you using?

I used prettier with the configuration mentioned in #72. I could revert it if needed.

**Pull request merged**